data/obaidashfaq20.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Obaid Ashfaq",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "a Ruby developer who is enthusiastic about writing clean code",
+  "avatar": "https://github.com/obaidashfaq20.png"
```

Comment:
Hi, please put a comma at the end of the line. So it won't have a syntax error in the json format.